src/components/Sheet/README.md

```diff
@@ -0,0 +1,280 @@
+---
+name: Sheet
+category: Overlays
+platforms:
+  - web
+keywords:
+  - sheet
+  - modal
+  - open
+  - title
+  - overlay
+  - drawer
+  - dialog
+  - web
+fullSizeExamples: true
+---
+
+# Sheet
+
+A sheet is a large container that enters from the edge of the screen when triggered by the merchant. It’s used to provide merchants with actions and information contextual to the page. It doesn’t interrupt their flow like a modal.
+
+---
+
+## Required components
+
+The sheet component must be wrapped in the [frame](/components/structure/frame) component.
+
+---
+
+## Use in an embedded application
+
+Use of the sheet component in an embedded application is not currently supported. If this is a feature you would like to see supported by Shopify App Bridge, let us know in the [forums](https://community.shopify.com/c/Shopify-APIs-SDKs/bd-p/shopify-apis-and-technology).
+
+---
+
+## Accessibility
+
+Sheets provide an opportunity to let merchants dig into more detail on their current task, or access information for their current task in a different way. Although merchants may be able to see content in the sheet and the main page content at the same time, they should only be expected to interact with one or the other at any given time.
+
+### Keyboard support
+
+- Use the `onClose` prop so that the sheet can be closed with the <kbd>esc</kbd> key as well as with button-based controls
+- Use a button to open the sheet
+- When the sheet opens, move focus to it so merchants who rely on the keyboard and screen readers can access it
```

Comment:
```suggestion
- When the sheet opens, move focus to it so merchants who rely on the keyboard and screen readers can access it
- Keep focus in the sheet until it is dismissed
```

I realize I left this out in the sheet draft, but we included it in the original list filters spec.